scripts/component_updation/main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/layer5io/component_scraper/pkg"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+)
+
+var (
+	ColumnNamesToExtract = []string{"Project Name", "Helm Chart", "Category", "Sub-Category", "Shape", "Primary Color", "Secondary Color", "Logo URL", "SVG_Color", "SVG_White"}
```

Comment:
@Revolyssup Please add a note in the spreadsheet that identifies the fact that columns are not to be renamed in the sheet without updating the code here.

Reply:
We might want to incorporate use of an email on failure of this workflow.